scripts/dataframe_analysis.py

```diff
@@ -185,6 +189,22 @@ def commodity_to_prototype(transactions_df, commodity, prototype):
     return prototype_transactions
 
 
+def transactions_to_prototype(transactions_df, prototype):
+    '''
+    Finds all transactions to a prototype
+
+    Parameters:
+    -----------
+    transactions_df: DataFrame
+    prototype: str
+    '''
```

Comment:
```suggestion
    Parameters
    ----------
    transactions_df : DataFrame
    prototype : str
    
    Returns
    -------
    ??
    '''
```
Same comment

Reply:
Missed finished that doc string. Thanks for catching that.